### PR TITLE
Fixes numbers and adds rank to CSE, Adds the Rank Progression for CMO

### PR DESCRIPTION
**Pull request by Captiajoe**
## About The Pull Request
Fixes the actual times, changes ranks and timings on CSE
LTJG 0hrs-25hrs
LT 25hrs-50hrs
LCDR 50hrs-100hrs
CDR 100hrs-1000hrs
CO 1000hrs on

Adds the missing rank progression for CMOs
LT 0hrs-50hrs
LCDR 50hrs-100hrs
CDR 100hrs-1000hrs
CO 1000hrs on

## Why It's Good For The Game
The CSE in the code had the time stamped to be
1501 to 6000 // 25hrs rank "03"
6001 to INFINITY // 50hrs rank "04"
in reality that is 25hrs to 100hrs for "03" and 100hrs to infinity for "04"
also added in the 1000hrs joke as the marines have

And the CMO has no rank progression at all, I added in the ranks from LT to CDR and added in the CO 1000hrs as marines have the 1000 hrs joke for SCPL

## Changelog
:cl:
add: Added new mechanics or gameplay changes
add: Added more mechanics or gameplay changes
expansion: Expands content of an existing thing
del: Removed old things
qol: made something easier to use
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

**Comment by Captiajoe** [comment]
i have no fucking clue if i did this right but i want my CMO ranks god damnit

**Comment by SpaceLoveSs13** [comment]
Cassie the captain rank changes were already proposed once by Nani ans tivi steictly said only 3 ranks, maximum 4. And about other ranks it was 3 ranks too. Now it can be changed obviously but that needs a good reasoning. Your PR is extremely incomplete, please provide a proper reasoning about why it's good for the game.

**Comment by Captiajoe** [comment]
> Cassie the captain rank changes were already proposed once by Nani ans tivi steictly said only 3 ranks, maximum 4. And about other ranks it was 3 ranks too. Now it can be changed obviously but that needs a good reasoning. Your PR is extremely incomplete, please provide a proper reasoning about why it's good for the game.

i have no idea how github works so ill see if i can change it around

File: code/datums/jobs/job/shipside.dm [hunk 1]
```diff
@@ -412,12 +416,16 @@ If you are not piloting, there is an autopilot fallback for command, but don't l
...
+		if(0 to 1500) // starting to 25 hrs LTJG
+			new_human.wear_id.paygrade = "O2" // LTJG
+		if(1501 to 3000) // 25 hrs to 50 hrs LT
+			new_human.wear_id.paygrade = "O3" // LT
+		if(3001 to 6000) // 50 hrs to 100 hrs LCDR
+			new_human.wear_id.paygrade = "O4" // LCDR 
+		if(6001 to 60000) // 100 hrs to 1000 hrs CDR
+			new_human.wear_id.paygrade = "05" // CDR 
+		if(60001 to INFINITY) // 1000 hrs holy fuck CO
+			nwe_human.wear_id.paygrade = "06" // CO
```
Comment (SpaceLoveSs13) [inline code/datums/jobs/job/shipside.dm:428]: Change the var to new, typo here is nwe.

**Review by SpaceLoveSs13 — CHANGES_REQUESTED** [review]

File: code/datums/jobs/job/shipside.dm [hunk 2]
```diff
@@ -425,7 +421,7 @@ If you are not piloting, there is an autopilot fallback for command, but don't l
 		if(6001 to 60000) // 100 hrs to 1000 hrs CDR
 			new_human.wear_id.paygrade = "05" // CDR 
 		if(60001 to INFINITY) // 1000 hrs holy fuck CO
-			nwe_human.wear_id.paygrade = "06" // CO
+			new_human.wear_id.paygrade = "06" // CO
```
Comment (SpaceLoveSs13) [inline code/datums/jobs/job/shipside.dm:424]: 👍

**Review by SpaceLoveSs13 — APPROVED** [review]

**Comment by Captiajoe** [comment]
i have no idea how to fix anything else wrong with how i set this github request up ;~;

**Comment by SpaceLoveSs13** [comment]
> i have no idea how to fix anything else wrong with how i set this github request up ;~;

Click the three dots beside ghe pr description, edit then just remove the tags that you don't want and their comments and keep yours. A maintainer will fix the tags once they see it. Don't worry.

**Review by SplinterGP — APPROVED** [review]

**Comment by TiviPlus** [comment]
I said it once I said it twice and now I saw it thrice:
NO NEW RANKS

**Pull request closed without merging**